### PR TITLE
Add spreadsheet config element

### PR DESCRIPTION
**Pull request by achour94**

### CODE REVIEW

File: src/translations/fr.json [hunk 1]
```diff
@@ -132,8 +133,8 @@
     "download.fileName": "Nom de fichier",
     "download.export.button": "Convertir",
     "download.exportFormat": "Format de l'export",
-    "download.message.downloadedCase": "Une situation a été téléchargée.",
-    "download.message.downloadedCases": "{number} situations ont été téléchargées.",
+    "download.message.downloadedElement": "Un élément a été téléchargée.",
```
Comment (FranckLecuyer) [inline src/translations/fr.json]: Typo : téléchargé

File: src/translations/fr.json [hunk 2]
```diff
@@ -132,8 +133,8 @@
...
+    "download.message.downloadedElements": "{number} éléments ont été téléchargées.",
```
Comment (FranckLecuyer) [inline src/translations/fr.json]: Typo: téléchargés

File: src/components/menus/directory-tree-contextual-menu.tsx [hunk 3]
```diff
@@ -172,6 +173,11 @@ const DirectoryTreeContextualMenu: React.FC<DirectoryTreeContextualMenuProps> =
                         handlePasteError(error);
                     });
                     break;
+                case ElementType.SPREADSHEET_CONFIG:
```
Comment (FranckLecuyer) [inline src/components/menus/directory-tree-contextual-menu.tsx:173]: You should also add the SPREADSHEET_CONFIG element type in the not clickable elements in getClickableRowStyle function in directory-content-table.tsx

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
Tests: 
Missing equipement type in parenthesis after 'Spreadsheet model' in column 'Type'
Maybe prettify the json result when downloading a spreadsheet_config ?

File: src/components/menus/content-contextual-menu.tsx [hunk 4]
```diff
@@ -444,21 +451,26 @@ const ContentContextualMenu = (props: ContentContextualMenuProps) => {
...
-                selectedElements[0].type === ElementType.CONTINGENCY_LIST ||
-                selectedElements[0].type === ElementType.FILTER ||
-                selectedElements[0].type === ElementType.MODIFICATION ||
-                selectedElements[0].type === ElementType.VOLTAGE_INIT_PARAMETERS ||
-                selectedElements[0].type === ElementType.SECURITY_ANALYSIS_PARAMETERS ||
-                selectedElements[0].type === ElementType.SENSITIVITY_PARAMETERS ||
-                selectedElements[0].type === ElementType.SHORT_CIRCUIT_PARAMETERS ||
-                selectedElements[0].type === ElementType.LOADFLOW_PARAMETERS)
-        );
+    const allowsDuplicateAndCopy = useCallback(() => {
```
Comment (AbdelHedhili) [inline src/components/menus/content-contextual-menu.tsx:454]: why this rename, isn't copying and duplicating the same thing

**Review by FranckLecuyer — APPROVED** [review]

**Review by AbdelHedhili — APPROVED** [review]

**Pull request merged**